### ISSUE TITLE
Display material name instead of ID

### ISSUE THREAD

**Issue by zentol** [Synthlight/MHW-Editor#4]
Material columns (like for .eq_crt files) currently display the ID of the material. This makes it a bit unwieldy to use, particularly so if one is trying to match the crafting requirements to the actual weapon/armor/etc.

I found the `*itemData.json` assets and the `DataHelper`, but couldn't figure out how to use it for a column. So I generated a crude Material enum from the assets that one could use just like the `AugmentationCategory`:
```
public enum CraftingMaterial
    {
        Unknown = 0 / 2,
        Potion = 2 / 2,
        Mega_Potion = 4 / 2,
        Max_Potion = 6 / 2,
        Ancient_Potion = 8 / 2,
        Antidote = 10 / 2,
        Herbal_Medicine = 12 / 2,
        Nulberry = 14 / 2,
        Energy_Drink = 16 / 2,
        Ration = 18 / 2,
        Rare_Steak = 20 / 2,
        Well_done_Steak = 22 / 2,
        ...
```

The IDs are halved for the same reason as in `Item` (the reason why I didn't just halve the values directly is because I suck at command-line text processing).

Do you have a better idea on how this could be done?

![exmple](https://user-images.githubusercontent.com/5725237/73609863-e921a880-45d1-11ea-9fcd-a23423a9886e.png)

**Comment by Synthlight** [comment]
Yea, the json asset files are more literal dumps from the GMD files made by the name extractor.
I can probably fix this in a few files I just hadn't gotten around to doing lookup lists for the more complicated things.
DataHelper loads all the info into dictionaries so the binding path would look like `DataHelper.itemData[locale][itemId] == item name`.
This however requires custom binding & sorting setup like for skills.
I'll try this later.

**Comment by Synthlight** [comment]
Should be done in the latest commit.
![image](https://user-images.githubusercontent.com/1075166/73612208-afdc3f00-45b7-11ea-91f6-c647ca9b51ff.png)
Not sure what list you're using for category though.

**Comment by Synthlight** [comment]
Done and released in v1.6.
